uhabits-android/src/main/java/org/isoron/uhabits/activities/habits/list/views/CheckmarkButtonView.kt

```diff
@@ -88,25 +89,28 @@ class CheckmarkButtonView(
         setOnLongClickListener(this)
     }
 
-    fun performToggle() {
+    fun performToggle(v: View) {
```

Comment:
Same remark about this `v` variable, make it more explicit so that it's easier to understand.